packages/ui/src/components/va-tree-view/VaTreeView.demo.vue

```diff
@@ -1,59 +1,340 @@
 <template>
   <VbDemo>
-    <VbCard>
-      <va-button @click="$refs.treeView.expand()">
-        EXPAND ALL
-      </va-button>
-      <va-button @click="$refs.treeView.collapse()">
-        COLLAPSE ALL
-      </va-button>
-
-      <va-tree-view ref="treeView">
-        <va-tree-category
-          label="Category text"
-          is-open
-        >
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-          <va-tree-category
-            label="Category text"
-            is-open
-          >
-            <va-tree-node>
-              Node text
-            </va-tree-node>
-          </va-tree-category>
-        </va-tree-category>
-      </va-tree-view>
+    <VbCard title="Default">
+      <va-tree-view :nodes="nodes" />
+    </VbCard>
+    <VbCard title="Stateless">
+      <va-tree-view :nodes="nodes" :stateful="false" />
+    </VbCard>
+    <VbCard title="Selectable">
+      <p>Default</p>
+      <va-tree-view :nodes="nodes" selectable />
+      <p>Selection strategy</p>
+      <va-radio
+        v-for="(option, index) in selectionTypeOptions"
+        :key="index"
+        v-model="selectionType"
+        :option="option"
+      />
+      <va-tree-view
+        :nodes="nodes"
+        :selection-type="selectionType"
+        selectable
+      />
+    </VbCard>
+    <VbCard title="Colored checkboxes">
+      <va-color-palette
+        v-model="selectedColor"
+        :palette="colorsPalette"
+      />
+      <va-tree-view
```

Comment:
<img width="229" alt="image" src="https://user-images.githubusercontent.com/5394573/191758795-07d637c3-dfd7-4a2c-a006-fbb0674c3b7c.png">

This demo could have all checkboxes set, so that we see instantly it's about color.